x-pack/legacy/plugins/maps/public/layers/fields/es_agg_field.js

```diff
@@ -69,4 +68,13 @@ export class ESAggMetricField extends AbstractField {
     }
     return metricAggConfig;
   }
+
+  supportsFieldMeta() {
+    // count and sum aggregations are not within field bounds so they do not support field meta.
+    return ![METRIC_TYPE.COUNT, METRIC_TYPE.SUM, METRIC_TYPE.UNIQUE_COUNT].includes(this.getAggType());
```

Comment:
minor imo, but this functional indirection is elaborate. simple conditional with ` || ` is more simple.

fwiw, this same check exists here as well https://github.com/elastic/kibana/blob/e2dc9213ca14e4dd090a0b9c7f603532475d6c38/x-pack/legacy/plugins/maps/public/layers/sources/es_geo_grid_source/update_source_editor.js#L75. 

Maybe extract to `isMetricCountable` utility function?